From fcd5d961820837962a3399b84f28e80661291627 Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Tue, 27 Aug 2013 14:49:59 +0200
Subject: [PATCH 9/9] ALSA: hda - Simplify CONFIG_SND_HDA_I915 condition

commit: 1c9a341bbdc14051a4d8c74ea67269786c7d3736 upstream

CONFIG_SND_HDA_I915 doesn't have to be user-selectable as this is almost
mandatory when i915 driver is available.  Let's enable it always when
CONFIG_DRM_I915 is set, so that user won't be bothered by useless
questions.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 sound/pci/hda/Kconfig |    9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/sound/pci/hda/Kconfig b/sound/pci/hda/Kconfig
index c5a872c..636e50c 100644
--- a/sound/pci/hda/Kconfig
+++ b/sound/pci/hda/Kconfig
@@ -153,14 +153,9 @@ config SND_HDA_CODEC_HDMI
 	  This module is automatically loaded at probing.
 
 config SND_HDA_I915
-	bool "Build Display HD-audio controller/codec power well support for i915 cards"
+	bool
+	default y
 	depends on DRM_I915
-	help
-	  Say Y here to include full HDMI and DisplayPort HD-audio controller/codec
-	  power-well support for Intel Haswell graphics cards based on the i915 driver.
-
-	  Note that this option must be enabled for Intel Haswell C+ stepping machines, otherwise
-	  the GPU audio controller/codecs will not be initialized or damaged when exit from S3 mode.
 
 config SND_HDA_CODEC_CIRRUS
 	bool "Build Cirrus Logic codec support"
-- 
1.7.9.5

